### PR TITLE
Added docs for $resolved and changelog for upcoming 2.1.0 release

### PR DESCRIPTION
**Pull request by jwngr**

### CODE REVIEW

File: docs/reference.md [hunk 1]
```diff
@@ -288,6 +290,27 @@ unwatch();
 Calling this method cancels event listeners and frees memory used by this object (deletes the
 local data). Changes are no longer synchronized to or from the database.
 
+### $resolved
+
+Returns the loaded state for this object. Returns `true` if the initial object data has been
+downloaded from the database; otherwise, returns `false`.
```
Comment (katowulf) [inline docs/reference.md]: More specifically, I'd like the relationship to $loaded() to be defined here.

File: docs/reference.md [hunk 2]
```diff
@@ -547,6 +570,27 @@ function compare(a, b) {
 Stop listening for events and free memory used by this array (empties the local copy).
 Changes are no longer synchronized to or from the database.
 
+### $resolved
+
+Returns the loaded state for this array. Returns `true` if the initial array data has been
+downloaded from the database; otherwise, returns `false`.
```
Comment (katowulf) [inline docs/reference.md]: Same here.

**Comment by katowulf** [comment]
Looks good! Two nitpicks.

**Comment by jwngr** [comment]
Updated the documentation. Please take another look.

File: docs/reference.md [hunk 3]
```diff
@@ -288,6 +290,32 @@ unwatch();
 Calling this method cancels event listeners and frees memory used by this object (deletes the
 local data). Changes are no longer synchronized to or from the database.
 
+### $resolved
+
+Attribute which represents the loaded state for this object. Its value will be `true` if the initial
+object data has been downloaded from the database; otherwise, its value will be `false`. This
+attribute is complementary to the `$loaded()` method. If the `$loaded()` promise is completed
```
Comment (katowulf) [inline docs/reference.md]: ♡

**Comment by katowulf** [comment]
lgtm

**Pull request merged**